src/ddb/ddb_entry.c

```diff
@@ -89,6 +89,32 @@ file_exists(const char *path)
 	return access(path, F_OK) == 0;
 }
 
+static int
+get_lines(const char *path, ddb_io_line_cb line_cb, void *cb_args)
+{
+	FILE	*f;
+	char	*line = NULL;
+	uint64_t len = 0;
+	uint64_t read;
```

Comment:
readline() can return negative value on error.

Reply:
getline returns a size_t

Reply:
Yes, according to the man page, its return type is "size_t", but it also said "Both functions return -1 on failure to read a line (including end-of-file condition)." Just thinking whether "uint64_t != -1" is good practice or not.

Reply:
Oh, right. Actually the while loop exits when getline returns -1 because -1 is returned on end of file condition. Added a check of errno after while loop exits.